src/rivescript.coffee

```diff
@@ -392,27 +392,35 @@ class RiveScript
 
   # Load a file using ajax. DO NOT CALL THIS DIRECTLY.
   _ajaxLoadFile: (loadCount, file, onSuccess, onError) ->
-    # Make the ajax request with jQuery. TODO: don't use jQuery.
-    $.ajax
-      url: file
-      dataType: "text"
-      success: (data, textStatus, xhr) =>
-        @say "Loading file #{file} complete."
-
-        # Parse it!
-        @parse file, data, onError
-
-        # Log that we've received this file.
-        delete @_pending[loadCount][file]
-
-        # All gone?
-        if Object.keys(@_pending[loadCount]).length is 0
-          if typeof(onSuccess) is "function"
-            onSuccess.call undefined, loadCount
-      error: (xhr, textStatus, errorThrown) =>
-        @say "Ajax error! #{textStatus}; #{errorThrown}"
-        if typeof(onError) is "function"
-          onError.call undefined, textStatus, loadCount
+
+    self = @
```

Comment:
In CoffeeScript, you can avoid needing to do `self = @` by using the wide arrow operator for the nested functions (e.g. on line 399, `xhr.onreadystatechange = () =>`). [More information here](http://coffeescript.org/#fat-arrow)

Reply:
@kirsle thanks for the review i will make my changes now.